insights/parsers/dirsrv_logs.py

```diff
@@ -0,0 +1,95 @@
+'''
+389 Directory Server logs
+=========================
+
+'''
+
+from insights import parser, LogFileOutput
+
+
+@parser('dirsrv_access')
```

Comment:
@PaulWay you use the spec `dirsrv_access` here, and you added the spec `dirsrv_logs` in the specs, which one should it be?